### DISCUSSION TITLE
Alguém estuda FIAP EAD?

### DISCUSSION THREAD

**Discussion by bartier** [frontendbr/forum#1626] [Discussão]
Pessoal, tudo bem?

Alguém faz o curso de Técnologo em Desenvolvimento de Sistemas ou Sistemas de Informação na FIAP?

Ambos cursos tem a opção EAD e como já estou na 'área' acho que seria uma boa opção já que poderia otimizar meu tempo e tudo mais...

[Sistemas de Informação EAD - Bacharelado](https://www.fiap.com.br/online/graduacao/bacharelado/sistemas-de-informacao/)

[ADS - Tecnólogo](https://www.fiap.com.br/online/graduacao/tecnologo/analise-e-desenvolvimento-de-sistemas/)

Aproveitando a discussão, gostaria de saber se vocês acham que faz muita diferença cursar Tecnológo/Bacharel pensando especificamente no processo de imigração do Canadá, alguém teria experiência para contar? 

Desde já obrigado =)

**Comment by felipefialho** [comment]
Seguem links: #564 #1441

**Comment by Levis44** [comment]
Cara, eu estou pensando em fazer Sistemas de Info. EAD. Vc começou a fazer? o que tem a dizer?

**Reply by bartier** [reply]
@Levis44 Cara, desculpa a demora para responder aqui. Não cheguei a fazer FIAP EAD, estou postergando faculdade rs. Por conta disso não tenho como dar uma visão a respeito. Você por acaso chegou a fazer lá?

**Reply by Levis44** [reply]
@bartier  hahahah relaxa mano! Me matriculei, vou começar semana que vem!

**Reply by JadeFatturi** [reply]
@Levis44 Eu também to querendo fazer e queria saber a tua opinião com a FIAP EAD. Cara que dúvida mortal. (até porque o investimento é alto e não quero ser uma decepção).

**Reply by Levis44** [reply]
@JadeFatturi Olha, depois de mais de 1 ano de curso, posso falar que estou bem decepcionado kkkk. Não aprendi muita coisa, o EAD é bem jogado, com atividades meio sem sentido e mal organizado. Só não troco para o presencial por conta do trabalho. Se você quer realmente aprender, te indico o presencial.

**Reply by JadeFatturi** [reply]
Puts, não acredito! Início do ano comecei em uma faculdade, mas detestei a forma que o conteúdo era jogado, tenho foco em fazer algo no EAD mesmo ;/ De qualquer forma obrigada!

**Reply by Levis44** [reply]
Pois é, também me decepcionei bastante hahaha. De qualquer forma, precisando só dar um toque!
